### PR TITLE
Adding domain specific examples

### PR DESCRIPTION
**Pull request by NivekT**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #219
* __->__ #216
* #212

Differential Revision: [D34174183](https://our.internmc.facebook.com/intern/diff/D34174183)

### CODE REVIEW

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34174183).

**Review by ejguan — APPROVED** [review]
Thanks a lot. This looks awesome

File: docs/source/examples.rst [hunk 1]
```diff
@@ -3,19 +3,64 @@ Examples
...
+
 Text
 -----------
 
-Audio
+CC-100
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+`CC-100 : Monolingual Datasets from Web Crawl Data <http://data.statmt.org/cc-100/>`_ is a corpus comprises of
+monolingual data for 100+ languages and also includes data for romanized languages. You can find the DataPipe
+implementation in `this Jupyter Notebook <https://github.com/pytorch/data/blob/main/examples/text/CC100.ipynb>`_.
```
Comment (ejguan) [inline docs/source/examples.rst:27]: BTW: Could you remove this CC-100? I haven't tested it for a while as we have re-implemented a few DataPipes.

File: docs/source/examples.rst [hunk 2]
```diff
@@ -3,19 +3,64 @@ Examples
...
+
+
+IMDB
```
Comment (ejguan) [inline docs/source/examples.rst:30]: And, all TorchText Datasets are fully revamped in TorchText repo here: https://github.com/pytorch/text/tree/main/torchtext/datasets

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34174183).

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34174183).

**Review by VitalyFedyunin — APPROVED** [review]

**Pull request closed without merging**